lib/datadog/profiling/exporter.rb

```diff
@@ -75,6 +80,8 @@ def flush
           internal_metadata: internal_metadata.merge(
             {
               worker_stats: worker_stats,
+              profile_stats: profile_stats,
+              recorder_stats: pprof_recorder.stats,
```

Comment:
Minor: I wonder if we should just return the recorder stats from the `serialize` as well?

Reply:
Feels a bit icky from an API perspective since they are kinda separate from the profile itself?